### PR TITLE
bumping up threshhold

### PR DESCRIPTION
**Pull request by kammerdiener**

### CODE REVIEW

File: infra/ceramic-mainnet/index.ts [hunk 1]
```diff
@@ -223,7 +223,7 @@ export const frontendUrlEcs = pulumi.interpolate`http://${httpListener.endpoint.
 const target = alb.createTargetGroup("gitcoin-dpopp-ceramic", {
   vpc,
   port: 80,
-  healthCheck: { path: "/api/v0/node/healthcheck" },
+  healthCheck: { path: "/api/v0/node/healthcheck", unhealthyThreshold: 3, port: "8011", interval: 60, timeout: 30, healthyThreshold: 2 },
```
Comment (smrz2001) [inline infra/ceramic-mainnet/index.ts:226]: Should this be port `80` instead of `8011`?

Comment (smrz2001) [reply infra/ceramic-mainnet/index.ts:226]: Or just not need to be specified?

Comment (smrz2001) [reply infra/ceramic-mainnet/index.ts:226]: We have it set to `7007` since we use the default Ceramic port.

**Review by smrz2001 — APPROVED** [review]
LGTM 🚀

**Review by nutrina — APPROVED** [review]
LGTM

**Pull request merged**